elements/pfe-tooltip/BaseTooltip.scss

```diff
@@ -0,0 +1,133 @@
+        These are popper specific data attributes.
+**/
+
+[data-popper-placement='top'] .arrow,
+[data-popper-placement='top-end'] .arrow,
+[data-popper-placement='top-start'] .arrow {
+  bottom: 0;
+  left: 50%;
+  transform:
+    translateX(var(--pf-c-tooltip__arrow--m-top--TranslateX, -50%))
+    translateY(var(--pf-c-tooltip__arrow--m-top--TranslateY, 50%))
+    rotate(var(--pf-c-tooltip__arrow--m-top--Rotate, 45deg));
+}
+
+[data-popper-placement='bottom'] .arrow,
+[data-popper-placement='bottom-end'] .arrow,
+[data-popper-placement='bottom-start'] .arrow {
+  top: 0;
+  left: 50%;
+  transform:
+    translateX(var(--pf-c-tooltip__arrow--m-bottom--TranslateX, -50%))
+    translateY(var(--pf-c-tooltip__arrow--m-bottom--TranslateY, -50%))
+    rotate(var(--pf-c-tooltip__arrow--m-bottom--Rotate, 45deg));
+}
+
+[data-popper-placement='left'] .arrow,
+[data-popper-placement='left-end'] .arrow,
+[data-popper-placement='left-start'] .arrow {
+  top: 50%;
+  right: 0;
+  transform:
+    translateX(var(--pf-c-tooltip__arrow--m-left--TranslateX, 50%))
+    translateY(var(--pf-c-tooltip__arrow--m-left--TranslateY, -50%))
+    rotate(var(--pf-c-tooltip__arrow--m-left--Rotate, 45deg));
+}
+
+[data-popper-placement='right'] .arrow,
+[data-popper-placement='right-end'] .arrow,
+[data-popper-placement='right-start'] .arrow {
+  top: 50%;
+  left: 0;
+  transform:
+    translateX(var(--pf-c-tooltip__arrow--m-right--TranslateX, -50%))
+    translateY(var(--pf-c-tooltip__arrow--m-right--TranslateY, -50%))
+    rotate(var(--pf-c-tooltip__arrow--m-right--Rotate, 45deg));
+}
+
+[data-popper-placement='left-start'] .arrow,
+[data-popper-placement='right-start'] .arrow {
+  top: var(--pf-c-tooltip__arrow--Height, 0.5rem);
+}
+
+[data-popper-placement='left-end'] .arrow,
+[data-popper-placement='right-end'] .arrow {
+  top: auto;
+  bottom: 0;
+}
+
+[data-popper-placement='top-start'] .arrow,
+[data-popper-placement='bottom-start'] .arrow {
+  left: var(--pf-c-tooltip__arrow--Width, 0.5rem);
+}
+
+[data-popper-placement='top-end'] .arrow,
+[data-popper-placement='bottom-end'] .arrow {
+  right: 0;
+  left: auto;
+}
+
+.content {
+  position: relative;
+  padding:
+    var(--pf-c-tooltip__content--PaddingTop,
+      var(--pf-global--spacer--sm, 0.5rem))
+    var(--pf-c-tooltip__content--PaddingRight,
+      var(--pf-global--spacer--sm, 0.5rem))
+    var(--pf-c-tooltip__content--PaddingBottom,
+      var(--pf-global--spacer--sm, 0.5rem))
+    var(--pf-c-tooltip__content--PaddingLeft,
+      var(--pf-global--spacer--sm, 0.5rem));
+  font-size: var(--pf-c-tooltip__content--FontSize,
+    var(--pf-global--FontSize--sm, 0.875rem));
+  color: var(--pf-c-tooltip__content--Color, var(--pf-global--Color--light-100, #ffffff));
+  text-align: center;
+  word-break: break-word;
+  background-color: var(--pf-c-tooltip__content--BackgroundColor,
+    var(--pf-global--BackgroundColor--dark-100, #1b1d21));
```

Comment:
@brianferry, thanks for making the bg color update for #tooltip. Does it have to be changed in .content and .arrow to show up too?

Reply:
Yes, my bad.  Updated.

Reply:
Awesome, thanks! The button demos look good too.

Reply:
Awesome @marionnegp , if everything looks good would you be so kind as to give an approval on the MR?